src/app/games/teeko/tests/teeko.component.spec.ts

```diff
@@ -0,0 +1,236 @@
+/* eslint-disable max-lines-per-function */
+import { fakeAsync } from '@angular/core/testing';
+import { ComponentTestUtils } from 'src/app/utils/tests/TestUtils.spec';
+import { TeekoComponent } from '../teeko.component';
+import { PlayerOrNone } from 'src/app/jscaip/Player';
+import { Table } from 'src/app/utils/ArrayUtils';
+import { TeekoState } from '../TeekoState';
+import { RulesFailure } from 'src/app/jscaip/RulesFailure';
+import { TeekoDropMove, TeekoMove, TeekoTranslateMove } from '../TeekoMove';
+import { Coord } from 'src/app/jscaip/Coord';
+
+describe('TeekoComponent', () => {
+
+    const _: PlayerOrNone = PlayerOrNone.NONE;
+    const O: PlayerOrNone = PlayerOrNone.ZERO;
+    const X: PlayerOrNone = PlayerOrNone.ONE;
+
+    let testUtils: ComponentTestUtils<TeekoComponent>;
+
+    beforeEach(fakeAsync(async() => {
+        testUtils = await ComponentTestUtils.forGame<TeekoComponent>('Teeko');
+    }));
+    it('should create', () => {
+        testUtils.expectToBeCreated();
+    });
+    describe('drop phase', () => {
+        it('should toast error when clicking on occupied space', fakeAsync(async() => {
```

Comment:
should fail (not toast error, ça c'est expectMoveFailure qui s'en occupe, mais ce qui compte ici c'est que le move fail)

Reply:
x plusieurs